### ISSUE TITLE
App show error on IOS simulator

### ISSUE THREAD

**Issue by joelpanor** [Soomgo-Mobile/react-native-use-modal#5]
When I navigate back I have this error
![simulator_screenshot_3CBE5A2B-5A8D-4098-8DAF-3395A92062D2](https://user-images.githubusercontent.com/20626624/202508253-8e1c0bb2-521a-4d4b-968e-e8607da6a288.png)

In console I have this trace :
<img width="552" alt="Capture d’écran 2022-11-17 à 12 55 15" src="https://user-images.githubusercontent.com/20626624/202508700-bcfd8d87-deb8-492e-814e-fe8ecc93b2c5.png">

**Comment by chenyl4869** [comment]
same thing....

**Comment by Brune04** [comment]
I'm also running into this issue after upgrading to React Native 0.70.5. Is there any update on this?

**Comment by zeallat** [comment]
@Brune04 @chenyl4869 @joelpanor
Sorry for the late confirmation. Based on Brune04's comment, it seems like a problem with RN 0.70 and higher. I'll check it out quickly.

**Comment by zeallat** [comment]
@Brune04 @chenyl4869 @joelpanor
I haven't tested it, but I've released a version with this issue resolved as `v1.0.1`.

Perhaps the cause was that `DeviceEventEmitter#removeListener` was [removed in RN 0.70](https://github.com/facebook/react-native/blob/main/CHANGELOG.md#removed-1:~:text=Remove%20deprecated%20removeListener%20methods), but the old version of react-native-modal used in our library was using it. So I solved this problem by updating react-native-modal to the latest version. Thanks for the patience and let me know if the problem still seems unresolved.
